crates/iceberg/src/spec/mod.rs

```diff
@@ -38,3 +40,6 @@ pub use sort::*;
 pub use table_metadata::*;
 pub use transform::*;
 pub use values::*;
+pub use view_metadata::*;
+pub use view_version::*;
+// pub use view_metadata::*;
```

Comment:
We need to remove it or just uncomment it?

Reply:
https://github.com/c-thiel/iceberg-rust/commit/ae40cd515254872f1c2da4b0855373e9a7076540